py3status/modules/battery_level.py

```diff
@@ -53,15 +64,20 @@
 
 Obsolete configuration parameters:
     mode: an old way to define `format` parameter. The current behavior is:
-        if 'format' is specified, this parameter is completely ignored
-        if the value is `ascii_bar`, the `format` is set to `"{ascii_bar}"`
-        if the value is `text`, the `format` is set to `"Battery: {percent}"`
-        all other values are ignored
-        there is no default value for this parameter
-    show_percent_with_blocks: an old way to define `format` parameter:
-        if `format` is specified, this parameter is completely ignored
-        if the value is True, the `format` is set to `"{icon} {percent}%"`
-        there is no default value for this parameter
+        - if 'format' is not "{icon}", this parameter is completely ignored
+        - if 'format' is "{icon}" and 'mode' is "ascii_bar", the `format` is
+          set to "{ascii_bar}"
+        - if 'format' is "{icon}" and 'mode' is "text", the `format` is set to
+          "Battery: {percent}"
+        - all other values are ignored
+        default is None
+    show_percent_with_blocks: an old way to define `format` parameter. The
+      current behavior is:
+        - if 'format' is not "{icon}", this parameter is completely ignored
+        - if 'format' is "{icon}" and 'mode' is "ascii_bar" or "text", this
+          parameter is completely ignored
+        - if the value is True, the `format` is set to "{icon} {percent}%"
```

Comment:
Could you remove the `-` that you have added as it breaks the `modules/README.md`